2018.md

```diff
@@ -308,3 +308,10 @@ repo](https://github.com/theupdateframework/tuf).
 * Mentors: [Justin Cappos](https://github.com/JustinCappos),
   [Vladimir Diaz](https://github.com/vladimir.v.diaz), and
   [Sebastien Awwad](https://github.com/awwad)
+
+#### Conformance testing for Virtual Kubelet
```

Comment:
@rbitia Can you add a header for Virtual Kubelet like https://github.com/cncf/soc#prometheus or https://github.com/cncf/soc#open-policy-agent-opa? Thanks!

Reply:
Yes!